hack/jenkins/common.sh

```diff
@@ -429,17 +429,19 @@ if [ "$status" = "failure" ]; then
 fi
 echo "$description"
 
+REPORT_URL_BASE="https://storage.googleapis.com"
+
 if [ -z "${EXTERNAL}" ]; then
   # If we're already in GCP, then upload results to GCS directly
   SHORT_COMMIT=${COMMIT:0:7}
   JOB_GCS_BUCKET="minikube-builds/logs/${MINIKUBE_LOCATION}/${SHORT_COMMIT}/${JOB_NAME}"
-  echo ">> Copying ${TEST_OUT} to gs://${JOB_GCS_BUCKET}out.txt"
-  gsutil -qm cp "${TEST_OUT}" "gs://${JOB_GCS_BUCKET}out.txt"
-  echo ">> uploading ${JSON_OUT}"
+  echo ">> Copying ${TEST_OUT} to gs://${JOB_GCS_BUCKET}.out.txt to ${REPORT_URL_BASE}/${JOB_GCS_BUCKET}.out.txt"
```

Comment:
how about removing the Local Path from the Log it is hard to read
```
>> Copying /home/jenkins/minikube-integration/linux-amd64-kvm2-containerd-11794-394296-bed20c522b347651a293a5d4f30bffd0ce85267d/testout.txt to gs://minikube-builds/logs/11794/bed20c5/KVM_Linux_containerd.out.txt to https://storage.googleapis.com/minikube-builds/logs/11794/bed20c5/KVM_Linux_containerd.out.txt
```
change it to

```
completed with 1 / 262 failures in 85.71 minute(s).
>> Copying out report to https://storage.googleapis.com/minikube-builds/logs/11794/bed20c5/KVM_Linux_containerd.out.txt

```